easybuild/easyconfigs/k/Kraken2/Kraken2-2.0.8-beta-gompi-2019b-Perl-5.30.0.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'PackedBinary'
+
+name = 'Kraken2'
+version = '2.0.8-beta'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.ccb.jhu.edu/software/%(namelower)s/'
+description = """Kraken is a system for assigning taxonomic labels to short DNA sequences,
+ usually obtained through metagenomic studies. Previous attempts by other
+ bioinformatics software to accomplish this task have often used sequence
+ alignment or machine learning techniques that were quite slow, leading to
+ the development of less sensitive but much faster abundance estimation
+ programs. Kraken aims to achieve high sensitivity and high speed by
+ utilizing exact alignments of k-mers and a novel classification algorithm."""
+
+# part is compiled with $CXX, the rest is in Perl
+toolchain = {'name': 'gompi', 'version': '2019b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+github_account = 'DerrickWood'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-2.0.7-beta_CXX-CXXFLAGS.patch']
+checksums = [
+    'f2a91fc57a40b3e87df8ac2ea7c0ff1060cc9295c95de417ee53249ee3f7ad8e',  # v2.0.8-beta.tar.gz
+    '191291e51a846d193a12486ab84496d5dd221a05c84648688b1351bb84d7adb2',  # Kraken2-2.0.7-beta_CXX-CXXFLAGS.patch
+]
+
+dependencies = [
+    ('Perl', '5.30.0'),
+    ('BLAST+', '2.9.0'),
+]
+
+osdependencies = ['wget']
```

Comment:
@smoors why the switch to an OS dependency here?

Not a problem, just wondering why you changed it.

Reply:
@verdurin I figured it would be better to use the OS version which is more likely to get timely security updates.